openshift/inspectJob-template.yaml

```diff
@@ -86,3 +107,23 @@ parameters:
     description: Memory requested for build
     displayName: Build Memory
     default: 256Mi
+
+  - name: cpu_family
```

Comment:
Let's keep consistent, can be these values upper case as the rest of the parameters in templates?